src/component/CountryCards/CountryCardsContainer.jsx

```diff
@@ -0,0 +1,54 @@
+import { connect } from "react-redux";
+import CountryCards from "./CountryCards";
+import {
+  setCountries,
+  selectCardCountry,
+  deleteCountryCard,
+} from "../../redux/card-reduser";
+import SelectedCards from "../SelectedCards/SelectedCards";
+
+const CountryCardsContainer = (props) => {
+  const handleChange = (event) => {
+    return event.target.checked;
+  };
+
+  return (
+    <div>
+      {props.selectedCards.length !== 0 ? (
+        <SelectedCards
+          selectedCards={props.selectedCards}
+          handleChange={handleChange}
+          deleteCountryCard={props.deleteCountryCard}
+          // style={style}
+        />
+      ) : (
+        ""
+      )}
+      {props.isClear ? (
+        ""
+      ) : (
+        <CountryCards
+          countries={props.countries}
+          setCountries={props.setCountries}
+          searchWords={props.searchWords}
+          selectCardCountry={props.selectCardCountry}
+          selectedCards={props.selectedCards}
+        />
+      )}
+    </div>
+  );
+};
+
+const mapStateToProps = (state) => {
```

Comment:
https://levelup.gitconnected.com/react-redux-hooks-useselector-and-usedispatch-f7d8c7f75cdd
there is no point in using mapStateToProps as we have redux hooks